lib/lambda_ethereum_consensus/state_transition/epoch_processing.ex

```diff
@@ -0,0 +1,33 @@
+defmodule LambdaEthereumConsensus.StateTransition.EpochProcessing do
+  @moduledoc """
+  This module contains utility functions for handling epoch processing
+  """
+
+  alias ChainSpec
+  alias SszTypes.BeaconState
+
+  @spec process_eth1_data_reset(BeaconState.t()) :: BeaconState.t()
```

Comment:
Yeah, it makes sense.

I mainly did it this way because this function shouldn't fail, so I didn't see the need to add an exception handler and return an error.